contracts/account/manager/src/commands.rs

```diff
@@ -430,34 +433,40 @@ pub fn update_info(
     Ok(ManagerResponse::action("update_info"))
 }
 
-pub fn update_subscription_status(
+pub fn update_suspension_status(
     deps: DepsMut,
     info: MessageInfo,
-    new_status: Subscribed,
+    suspension_status: Subscribed,
+    response: Response,
 ) -> ManagerResult {
     let config = CONFIG.load(deps.storage)?;
 
     // Only the subscription contract can load
     if let Some(sub_addr) = config.subscription_address {
```

Comment:
so same here, i'm thinking of removing the subscription address from the contract's state.